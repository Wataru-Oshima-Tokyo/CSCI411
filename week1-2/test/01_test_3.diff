1,61c1,3
< Number of nodes and number of edges: 
< Add 24 edges: 
< 
< Original
< 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 
< 2 3 
< 1 6 
< 2 6 7 
< 5 11 
< 4 11 
< 12 13 
< 13 8 
< 9 11 
< 10 
< 7 15 
< 10 
< 14 
< 12 
< 13 
< 
<  Transposed 
< 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 
< 2 
< 1 3 
< 1 
< 5 
< 4 
< 2 3 
< 3 10 
< 7 
< 8 
< 9 11 
< 4 5 8 
< 6 13 
< 6 7 14 
< 12 
< 10 
< 0 0
< 1 0
< 2 0
< 3 0
< 4 0
< 5 0
< 6 0
< 7 0
< 8 0
< 9 0
< 10 0
< 11 0
< 12 0
< 13 0
< 14 0
< 15 1
< sccs size: 2
< id: 0 size: 14
< In: 0
< Out 1
< id: 1 size: 1
< In: 1
< Out 0
< |A| = 14, |B| = 1, |C| = 0
\ No newline at end of file
---
> Number of nodes and number of edges: 
> Add 24 edges: 
> |A| = 5, |B| = 4, |C| = 6
\ No newline at end of file
