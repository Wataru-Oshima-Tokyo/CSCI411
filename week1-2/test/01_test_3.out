Number of nodes and number of edges: 
Add 24 edges: 
|A| = 5, |B| = 4, |C| = 6